src/pybind/mgr/dashboard/frontend/src/app/ceph/dashboard-v3/dashboard/dashboard-v3.component.ts

```diff
@@ -44,15 +44,17 @@ export class DashboardV3Component extends PrometheusListHelper implements OnInit
   prometheusAlerts$: Observable<AlertmanagerAlert[]>;
 
   icons = Icons;
-  showAlerts = false;
   flexHeight = true;
   simplebar = {
     autoHide: false
   };
-  textClass: string;
+  alertClass: object = {
+    critical: 'danger',
+    warning: 'warning',
+    info: 'info'
+  };
```

Comment:
Why not using an Enum here (in general `object` types are not much welcome in Typescript, since they don't really enforce any typing)?